frontend/src/bundles/common/components/range-slider/range-slider.tsx

```diff
@@ -0,0 +1,60 @@
+import 'rc-slider/assets/index.css';
+import '../../../../assets/css/variables/color-variables.css';
+
+import Slider from 'rc-slider';
+import React, { useCallback, useState } from 'react';
+
+import styles from './styles.module.scss';
+
+interface RangeSliderProperties {
+    onChange?: (range: { min: number; max: number }) => void;
+    rangeLimits?: { min: number; max: number };
+}
+
+const RangeSlider: React.FC<RangeSliderProperties> = ({
+    onChange,
+    rangeLimits = { min: -100, max: 1000 },
+}) => {
+    const [range, setRange] = useState([rangeLimits.min, rangeLimits.max]);
```

Comment:
I think this state is redundant. You will have the same in the parent component.
So you have to add another prop, named like currentRange